src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SimpleConfigModel.cs

```diff
@@ -81,6 +81,8 @@ public SimpleConfigModel()
 
         public bool HasScriptRunningPostActions { get; set; }
 
+        public DateTime? ConfigTimestampUtc { get; set; }
```

Comment:
Since this isn't needed in IRunnableProjectConfig, it won't be needed here. 
This class deals with reading & processing the actual tempalte.json, as opposed to the reduced data in the cache.